ehrql/tables/beta/tpp.py

```diff
@@ -33,6 +33,20 @@
 
 @table
 class clinical_events(EventFrame):
+    """
+    Each record corresponds to a single clinical or consultation event for a patient.
+
+    Each record contains either a SNOMED-CT code or a CTv3 code (never both). Note that
+    date may be null.
```

Comment:
This is true, but I'm not sure it quite captures the weird way this table is structured.

I might say something like this:

> Each event is recorded twice: once with a CTv3 code, and again with the equivalent SNOMED-CT code. Each record will have only one of the `ctv3_code` or `snomedct_code` columns set and the other will be null. This allows you to query the table using either a CTv3 codelist or SNOMED-CT codelist and all records using the other coding system will be effectively ignored.

That's maybe not the clearest wording, but hopefully you see what I'm getting at.

Reply:
Ah TIL that I didn't understand the underlying tables anyway.

Reply:
I spent a while trying to figure out how to make the wording clearer, but I couldn't come up with anything better, so just going with your suggested wording :)